apps/full-site-editing/full-site-editing-plugin/full-site-editing/plugins/close-button-override/index.js

```diff
@@ -4,28 +4,42 @@
  * External dependencies
  */
 import domReady from '@wordpress/dom-ready';
+import { __ } from '@wordpress/i18n';
+import './style.scss';
 
 domReady( () => {
 	const { closeButtonLabel, closeButtonUrl, editorPostType } = fullSiteEditing;
 
 	const editPostHeaderInception = setInterval( () => {
-		const closeButton = document.querySelector( '.edit-post-fullscreen-mode-close__toolbar a' );
+		// cycle through interval until header toolbar is found
+		const toolbar = document.querySelector( '.edit-post-header__toolbar' );
 
-		if ( ! closeButton ) {
+		if ( ! toolbar ) {
 			return;
 		}
 		clearInterval( editPostHeaderInception );
 
+		// add components toolbar with override class name (original will be hidden in ./style.scss)
+		const componentsToolbar = document.createElement( 'div' );
+		componentsToolbar.className =
+			'components-toolbar edit-post-fullscreen-mode-close__toolbar edit-post-fullscreen-mode-close__toolbar__override';
+		toolbar.prepend( componentsToolbar );
```

Comment:
I wish there was a better way to handle this, but I don't think there is 🤔For other reviewers, this HTML element is entirely removed from the DOM when you are not full screen, so it needs to be manually re-added... The only way to get around this would be to add an extension point to Gutenberg. (Possibly something to look into)